content/hello-world-program.md

```diff
@@ -0,0 +1,328 @@
+# Hello World
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Describe the Solana account model
+- Explain the entry point to a Solana program
+- Submit a transaction to invoke our “Hello, world!” program
+
+# TL;DR
+
+- **Accounts** on Solana each have a unique address, store data, and are owned by a program
+- **Programs** on Solana are a particular type of account that is marked as executable and which stores instruction logic
+- **Transactions** on Solana are made up of instructions which are executed by programs
+
+# Overview
+
+In this lesson we'll be using the Rust programming language to write a Solana program. We will also be using the Solana Playground to deploy a Solana program.
```

Comment:
The first few paragraphs before the Account Model section here aren't cutting it for me. They seem disjointed somehow. I'd prefer something to introduce programs just before diving into the "in this lesson we will..." IDK exactly but something like:

> Solana's ability to run arbitrary executable code is part of what makes it so powerful. Solana programs, similar to "smart contracts" in other blockchain environments, are quite literally the backbone of the Solana ecosystem. And the collection of programs grows daily as developers and creators dream up and deploy new programs.

> This lesson will give you a basic introduction to writing a deploying a Solana program using the Rust programming language. To avoid the distraction of setting up a local development environment, we'll be using a browser-based IDE called [Solana Playground](beta.solpg.io).

Then I'd jump into the first section from there.

Reply:
Please feel free to play with that - it can be better I'm sure but that sorrt of flow is what I'm going for.

Reply:
replaced overview paragraphs with content from above